rclcpp/include/rclcpp/timer.hpp

```diff
@@ -108,6 +112,36 @@ class GenericTimer : public TimerBase
     cancel();
   }
 
+  void
+  execute_callback()
+  {
+    execute_callback_delegate<>();
+  }
+
+  // void specialization
+  template<typename CallbackT = FunctorT,
+  typename std::enable_if<
+    rclcpp::function_traits::same_arguments<CallbackT, VoidCallbackType>::value
+  >::type * = nullptr
```

Comment:
Your original indentation looks more readable. Maybe moving the first `typename` to a new line would prevent `uncrustify` from reformatting this code further.